docs/integrations/log_auditbeat_linux.md

```diff
@@ -0,0 +1,335 @@
+
+Auditbeat uses `modules` to collect audit information.
+By default, Auditbeat uses a configuration that’s tailored to the operating system where Auditbeat is running.
+
+Replace the configuration file `/etc/auditbeat/auditbeat.yml` by the following content:
+
+```bash
+########################## Auditbeat Configuration #############################
+
+# =========================== Modules configuration ============================
+auditbeat.modules:
+
+# The auditd module collects events from the audit framework in the Linux
+# kernel. You need to specify audit rules for the events that you want to audit.
+- module: auditd
+  resolve_ids: true
+  failure_mode: silent
+  backlog_limit: 8196
+  rate_limit: 0
+  include_raw_message: false
+  include_warnings: false
+
+  # Load audit rules
+  audit_rules: |
+    ## Example of audit rules here. Comment what is NOT needed
+    ## Executions.
+    -a always,exit -F arch=b64 -S execve,execveat -k exec
+
+    ## External access (warning: these can be expensive to audit).
+    -a always,exit -F arch=b64 -S accept,bind,connect -F key=external-access
+
+    ## Identity changes.
+    -w /etc/group -p wa -k identity
+    -w /etc/passwd -p wa -k identity
+    -w /etc/gshadow -p wa -k identity
+
+    ## Unauthorized access attempts.
+    -a always,exit -F arch=b64 -S open,creat,truncate,ftruncate,openat,open_by_handle_at -F exit=-EACCES -k access
+    -a always,exit -F arch=b64 -S open,creat,truncate,ftruncate,openat,open_by_handle_at -F exit=-EPERM -k access
+
+# The file integrity module sends events when files are changed (created,
+# updated, deleted). The events contain file metadata and hashes.
+- module: file_integrity
+  paths:
+  - /bin
+  - /usr/bin
+  - /sbin
+  - /usr/sbin
+  - /etc
+
+  scan_at_start: true
+  scan_rate_per_sec: 50 MiB
+  max_file_size: 100 MiB
+  hash_types: [sha1]
+
+  # Detect changes to files included in subdirectories. Disabled by default.
+  recursive: false
+
+- module: system
+  datasets:
+    - package   # Installed, updated, and removed packages
+  period: 2m
+
+- module: system
+  datasets:
+    - host      # General host information, e.g. uptime, IPs
+    - login     # User logins, logouts, and system boots.
+    - process   # Started and stopped processes
+    - socket    # Opened and closed sockets
+    - user      # User information
+
+
+  user.detect_password_changes: true
+
+# ================================== Outputs ===================================
+# ---------------------------- Elasticsearch Output ----------------------------
+output.elasticsearch:
+  enabled: false
+
+# -------------------------------- File Output ---------------------------------
+output.file:
+  enabled: true
+
+  # Configure JSON encoding
+  codec.json:
+    #pretty: false
+    # Configure escaping HTML symbols in strings.
+    #escape_html: false
+
+  # Path to the directory where to save the generated files.
+  path: "/tmp/auditbeat"

```

Comment:
We should probably write these logs in the `/var/log/` directory (for example in `/var/log/auditbeat/`). Shouldn’t we also provide a `logrotate` configuration for these files?